modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -281,3 +281,5 @@ properties:
   example2: "![En8lII21ZHrdIaINw0+mSA==]"
 ----
 
+Take into consideration that key sizes may depend on the Java Cryptography Extension (JCE) version. For example, from version 1.8.0_161, AES key size should be of 32 characters (256 bytes), instead of 16 for previous versions.
```

Comment:
```suggestion
Take into consideration that key sizes may depend on the Java Cryptography Extension (JCE) version. For example, since version 1.8.0_161, AES key size should be of at least 32 characters (256 bytes), instead of 16 for previous versions.
```